client/price-format.js

```diff
@@ -0,0 +1,9 @@
+export default function priceFormatter(price) {
+  const priceString = price.toString();
+
+  switch (priceString.length) {
+    case 3: return `$${priceString[0]}.${priceString.slice(1)}`;
+    case 4: return `$${priceString.slice(0, 2)}.${priceString.slice(2)}`;
+    case 5: return `$${priceString.slice(0, 3)}.${priceString.slice(3)}`;
+  }
```

Comment:
I recommend using `.toFixed` or `Intl.NumberFormat` for this rather than coupling your solution to the number of digits. If the business adds a $1,000 item to their inventory, the front end devs shouldn't have to change the code again.